dataset-spec/dataset-spec.md

```diff
@@ -52,28 +51,14 @@ The coordinate reference system of the values is WGS84 longitude/latitude.
 
 ### Provider Object
 
-The object provides information about a provider. A provider is any of the organizations that created or processed the content of the dataset and therefore influenced the data offered by this dataset.
+The object provides information about a provider. A provider is any of the organizations that captured or processed the content of the dataset and therefore influenced the data offered by this dataset. May also include information about the final storage provider hosting the data.
 
 | Field Name | Type   | Description                                                  |
 | ---------- | ------ | ------------------------------------------------------------ |
 | name       | string | **REQUIRED.** The name of the organization or the individual. |
+| type       | string | The type of provider. Any of `producer`, `processor` or `host`. |
```

Comment:
We should also probably explain a bit more on what 'producer', 'processor' and 'host' mean. Doesn't need to be in this table, could be just below or above. But just a bit of explanation to help people make a decision about which to use.

Reply:
Okay, I'll add explanation about it next week.

Reply:
Added the explanation. Please check whether it needs more explanation or not.